spec/lib/task-data/schemas/install-coreos-spec.js

```diff
@@ -16,7 +16,8 @@ describe(require('path').basename(__filename), function() {
         "installDisk": "/dev/sda",
         "comport": "ttyS0",
         "osType": "linux",
-        "rootPassword": "RackHD"
+        "rootPassword": "RackHD",
+        "ignitionScriptUri": "generic.json"
```

Comment:
if it's only for unit test, present 'generic.json' is fine, no need for a dummy script